infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -27,6 +27,16 @@ exports.tasks.push({
         'project:taskcluster:smoketest:abc/*' ],
     };
     assert.deepEqual(expandedRole.scopes, expectedScopes.scopes);
-    await auth.deleteRole(roleId);
+    const query = {limit: 1};
+    const anHourAgo = Date.now() - (1000*60*60);
+    while (1) {
+      const res = await auth.listRoles2(query);
+      res.roles[0].roleId.includes('project:taskcluster:smoketest:') === true && res.roles[0].lastModified > anHourAgo ? await auth.deleteRole(roleId) : null;
```

Comment:
Please use an `if` statement, rather than a ternary operator(.. `?` .. `:` ..).

Also, the logic is backward here; we want to delete roles *older* than an hour, not younger :)

Reply:
Yes. I will change the ternary operator to an `if `statement.